test/e2e/specs/editor/various/toolbar-roving-tabindex.spec.js

```diff
@@ -14,6 +14,9 @@ test.describe( 'Toolbar roving tabindex', () => {
 		await admin.createNewPost();
 		await editor.insertBlock( { name: 'core/paragraph' } );
 		await page.keyboard.type( 'First block' );
+
+		// Ensure the fixed toolbar option is off.
+		await editor.setIsFixedToolbar( false );
```

Comment:
Nit: Could we add a link to the original issue just to give confidence to others to remove this line if we ever resolve the root cause? 🙂

Reply:
I'll link this PR; it has more context.